### PR TITLE
Keep prototype scripts

### PR DESCRIPTION
**Pull request by ama-gi**

### CODE REVIEW

**Comment by ama-gi** [comment]
Currently all scripts are removed from the prototype html, this fixes that.
Important for example If you have a textarea with a [ckeditor](https://github.com/egeloen/IvoryCKEditorBundle) script attached to it.

**Comment by merk** [comment]
Thanks!

Please add a test to cover this change and move the functionality behind a configuration option on initialisation and good to merge.

**Comment by ama-gi** [comment]
Please review the proposed changes

File: Resources/public/js/collections.js [hunk 1]
```diff
@@ -84,7 +85,11 @@
             }
 
             var html = this._getPrototypeHtml($prototype, this.internalCount++),
+            if (this.options.keepScripts) {
+                $row = $($.parseHTML(html, document, true));
```
Comment (merk) [inline Resources/public/js/collections.js]: Could this be simplified to `$row = $($.parseHTML(html, document, this.options.keepScripts));` ?

**Comment by ama-gi** [comment]
Thanks! I simplified it like you suggested, after looking at the parseHTML() documentation.

**Comment by merk** [comment]
Thanks!

**Pull request closed without merging**